quobyte.go

```diff
@@ -168,3 +179,23 @@ func (client *QuobyteClient) SetTenant(tenantName string) (string, error) {
 
 	return response.TenantID, nil
 }
+
+// IsValidUUID Validates given uuid
+func IsValidUUID(uuid string) bool {
+	r := regexp.MustCompile("^[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-4[a-fA-F0-9]{3}-[8|9|aA|bB][a-fA-F0-9]{3}-[a-fA-F0-9]{12}$")
```

Comment:
If a tenants name matches this pattern the verification will fail. This may happen as some tools (e.g. in OpenStack) do use similar IDs for automatically created tenant names. In such a case the name is thought to be an UUID and things will fail afterwards.
I think we have to do something like:
1) Try using the value as a TenantID
2) If that failed because of an invalid TenantID error, lookup as a volume name and retry if a TenantID is returned for the given name.